server/members/views.py

```diff
@@ -195,3 +198,17 @@ def post(self, request, *args, **kwargs):
             return JsonResponse(
                 {"error": "Failed to upload file to storage"}, status=500
             )
+    
+class PasswordResetRequest(APIView):
+    permission_classes = [IsApiKey]
+
+    def post(self, request, *args, **kwargs):
+        discord_id = request.data.get("discord_id")
+        user = get_object_or_404(User, discord_id=discord_id)
+
+        token = default_token_generator.make_token(user)
+        uid = urlsafe_base64_encode(force_bytes(user.pk))
+
+        reset_url = f"https://interview.swecc.org/password-reset-confirm/{uid}/{token}/"
```

Comment:
can we make this work with local dev too?

Reply:
Ugh, yeah I'll add that later. Tired af and want to let sophia reset password.